### PR TITLE
make types requeried for MessageContract public

### PR DESCRIPTION
**Pull request by voronov-maxim**

### CODE REVIEW

File: src/CoreWCF.Primitives/src/CoreWCF/MessageBodyMemberAttribute.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@ namespace CoreWCF
 {
     // TODO: Make this public
     [AttributeUsage(ServiceModelAttributeTargets.MessageMember, Inherited = false)]
-    internal class MessageBodyMemberAttribute : MessageContractMemberAttribute
```
Comment (mconnew) [inline src/CoreWCF.Primitives/src/CoreWCF/MessageBodyMemberAttribute.cs]: Can you remove the TODO comment to make this public 😄?

File: src/CoreWCF.Primitives/src/CoreWCF/MessageHeaderAttribute.cs [hunk 2]
```diff
@@ -4,7 +4,7 @@ namespace CoreWCF
 {
     // TODO: Investigate making public
     [AttributeUsage(ServiceModelAttributeTargets.MessageMember, AllowMultiple = false, Inherited = false)]
-    internal class MessageHeaderAttribute : MessageContractMemberAttribute
+    public class MessageHeaderAttribute : MessageContractMemberAttribute
```
Comment (mconnew) [inline src/CoreWCF.Primitives/src/CoreWCF/MessageHeaderAttribute.cs]: Can you remove the TODO comment

File: src/CoreWCF.Primitives/src/CoreWCF/MessageContractMemberAttribute.cs [hunk 3]
```diff
@@ -4,7 +4,7 @@
 namespace CoreWCF
 {
     // TODO: Make this public
-    internal abstract class MessageContractMemberAttribute : Attribute
+    public abstract class MessageContractMemberAttribute : Attribute
```
Comment (mconnew) [inline src/CoreWCF.Primitives/src/CoreWCF/MessageContractMemberAttribute.cs]: Remove the TODO comment

**Comment by mconnew** [comment]
Would you be able to contribute a test using these attributes? If you look at the existing tests you should be able to see how to write an end-to-end test which uses these.

**Comment by mconnew** [comment]
Just to be clear, I don't require the tests to accept this PR, it's a polite request 😄. The comments do need to be removed though. If you aren't able to write the tests, could you open an issue for tests needing to be added and list all the types you are making public in the issue description.

**Comment by voronov-maxim** [comment]
I will write tests

**Comment by voronov-maxim** [comment]
Hi @mconnew, add test

**Comment by mconnew** [comment]
Thanks @voronov-maxim. As we don't have automated PR tests yet, I'm going to have to pull this locally to verify but everything looks good. I'll most likely get to that tomorrow.

**Pull request merged**